pkgs/tools/admin/pgadmin/default.nix

```diff
@@ -74,6 +75,17 @@ let
   # keep the scope, as it is used throughout the derivation and tests
   # this also makes potential future overrides easier
   pythonPackages = python3.pkgs.overrideScope (final: prev: rec {
+    # flask-security-too 4.1.5 is incompatible with flask-babel 3.x
+    flask-babel = prev.flask-babel.overridePythonAttrs (oldAttrs: rec {
+      version = "2.0.0";
+      src = fetchPypi {
+        inherit pname version;
+        sha256 = "f9faf45cdb2e1a32ea2ec14403587d4295108f35017a7821a2b1acb8cfd9257d";
+      };
+      nativeBuildInputs = [ ];
+      format = "setuptools";
+      outputs = [ "out" ];
```

Comment:
What was the error before you overwrite outputs?

Reply:
Well, due to the update of `flask-babel` there is now:
```
outputs = [
    "out"
    "doc"
  ];
```
and the override does not prove `doc`.
The error message is:
`error: builder for '/nix/store/pz3rhdsvc0m0s8in3pqafvbifblbimq4-python3.10-flask-babel-2.0.0.drv' failed to produce output path for output 'doc' at '/nix/store/pz3rhdsvc0m0s8in3pqafvbifblbimq4-python3.10-flask-babel-2.0.0.drv'`

Reply:
Didn't know that, then it is necessary.